curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md

```diff
@@ -33,11 +33,11 @@ Finally don't forget to give your image an <code>alt</code> text.
 ```yml
 tests:
   - text: Your page should have an image element.
-    testString: assert($("img").length > 0, 'Your page should have an image element.');
+    testString: assert($("img").length);
   - text: Your image should have a <code>src</code> attribute that points to the kitten image.
-    testString: assert(new RegExp("\/\/bit.ly\/fcc-relaxing-cat|\/\/s3.amazonaws.com\/freecodecamp\/relaxing-cat.jpg", "gi").test($("img").attr("src")), 'Your image should have a <code>src</code> attribute that points to the kitten image.');
-  - text: Your image element <strong>must</strong> have an <code>alt</code> attribute.
-    testString: assert(code.match(/alt\s*?=\s*?(\"|\').*(\"|\')/), 'Your image element <strong>must</strong> have an <code>alt</code> attribute.');
+    testString: assert(/^https:\/\/bit\.ly\/fcc-relaxing-cat$/i.test($("img").attr("src")));
+  - text: Your image element <strong>must</strong> have an <code>alt</code> attribute with a non-empty value.
```

Comment:
```suggestion
  - text: Your image element's <code>alt</code> attribute <strong>must</strong> not be empty.
```
Since we're fiddling with this line on the PR, I guess this is a better way to say what we're trying to